example/patches/react-native-test-app+4.1.4.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/react-native-test-app/android/android-manifest.js b/node_modules/react-native-test-app/android/android-manifest.js
```

Comment:
Upstream PR for this patch: https://github.com/microsoft/react-native-test-app/pull/2399